### PR TITLE
Style adjustments + number formatting

### PR DESCRIPTION
**Pull request by lucianosantana**
This PR contains: 
- some minor style adjustments for better integration with Asana's own style
- included a display number formatting to avoid [floating point precision issues](https://stackoverflow.com/questions/1458633/how-to-deal-with-floating-point-number-precision-in-javascript) and long decimals in general

**Before**
image

**After**
image

### CODE REVIEW

**Comment by igtm** [comment]
Thank you for your great contribution!
Looks good to me!

**Comment by igtm** [comment]
But I'd say that `font-weight: bold` is better in the point of view of visibility.

**Comment by lucianosantana** [comment]
Hi @igtm ! Thanks for the feedback. 

My reasoning was that the circular bubble was already enough to highlight the story points sum. Also, if you compare the style of the bubbles with and without bold font in the task details view, you'll notice how the font weight contrasts with the rest of Asana's style.

Bold Font
image

Normal Font
image

On the other hand, you'll notice I actually kept the bold font in the number of cards count, since it doesn't have a wrapping bubble (see screenshots in my first message).

Let me know your thoughts.

**Comment by igtm** [comment]
Thank you for your precise info!
I understand your thoughts.

I found good example of explaining my thoughts just now!
This is Github's badge style.

スクリーンショット 2019-07-08 23 42 30
スクリーンショット 2019-07-08 23 42 18
As you can see, this is `bold` style.

I just wanted to highlight the `number`, not the `bubble`.
Besides that, I was originally inspired by the [Scrum for Trello](https://chrome.google.com/webstore/detail/scrum-for-trello/jdbcdblgjdpmfninkoogcfpnkjmndgje) .
That also has the `bold` style badge. (I don't like the square badge, though)

スクリーンショット 2019-07-08 23 53 23

In short, I believe legibility of story points is more important than "Tone & Manner".

**Comment by lucianosantana** [comment]
@igtm
I believe font-weight 600 is a good compromise. I've update it on my last commit. Check the results :

Task Detail: 
image

Board View:
image

**Comment by igtm** [comment]
@lucianosantana OK! I'll merge this:) Thank you for your great contribution!
